Remotion/Web/Core/Res/Themes/NovaViso/HTML/WebButton.css

```diff
@@ -2,105 +2,57 @@
 
 button.webButton
```

Comment:
In terms of variable usage and styling: we should introduce new variables for all the stlying we do on the button. border color, hover colors, ...

These are defined here and then we have one class for `primary`, `secondary`, ... that sets the variables to the new colors. This way we do not have to repeat the actual css properties and it is possible for outer controls/consumers to restyle only the web button

Reply:
Ah, that's new. IE didn't do it like that. Okay then, no underline.

Reply:
This might be implemented later, but currently is out of scope.